test/config_file.c

```diff
@@ -303,7 +303,7 @@ bootname=slot0\n\
 [slot.child.0]\n\
 device=/dev/null\n\
 parent=parent.0\n\
-bootname=slot0\n";
+bootname=slotchild0\n";
```

Comment:
Shouldn't this be part of the previous commit?

Reply:
Ah, it could be, but these three commits are how I wrote it in order. I thought it was kind of interesting that the second commit shows that the test case from the earlier one is no longer a valid reproducer for this error message.